go/apps/http_api_nostream/resource.py

```diff
@@ -247,6 +247,13 @@ def handle_PUT_in_reply_to(self, request, payload, in_reply_to):
     @inlineCallbacks
     def handle_PUT_send_to(self, request, payload):
         user_account = request.getUser()
+        optout_store = self.worker.vumi_api.get_user_api(user_account) \
+            .optout_store
+        optout = yield optout_store.get_opt_out(
+            payload.get('to_addr_type', 'msisdn'), payload.get('to_addr'))
```

Comment:
Ah, after looking at https://github.com/praekelt/vumi/pull/895 it looks like we are doing that. Maybe we don't need this default here then?